log/sequencer.go

```diff
@@ -103,7 +101,6 @@ type Sequencer struct {
 	hasher     hashers.LogHasher
 	timeSource clock.TimeSource
 	logStorage storage.LogStorage
-	signer     *tcrypto.Signer
```

Comment:
Hm, was this field unused?

Reply:
Yes, the use was removed in this commit:

https://github.com/google/trillian/commit/884ef511ce166d1cb1ffa1ae2cb4aefb3018591d#diff-6074443f242fbef0f5f94ab179ab499f8ee83fedf0a36f79fbe52c637ca14cdd

Reply:
Also, it still builds! 😃